src/de.rs

```diff
@@ -54,14 +65,17 @@ impl<'de, 'a> serde::Deserializer<'de> for &'a mut Deserializer<'de> {
     }
 }
 
-
 struct Value {
     name: String,
     value: String,
 }
 
 impl Value {
-    fn parse<T>(&self) -> Result<T> where T: FromStr {
+    #[allow(clippy::map_err_ignore)]
```

Comment:
It's better to not ignore this error and make the parser error available in the caller instead of ignoring this problem...